python/tank/authentication/console_authentication.py

```diff
@@ -61,21 +63,29 @@ def authenticate(self, hostname, login, http_proxy):
                 # Insert a \n on the current line so the print is displayed on a new time.
                 print()
                 raise AuthenticationCancelled()
+            except ConsoleLoginWithSSONotSupportedError as e:
+                # SSO login requires a QtApplication environment at this time.
+                print("%s" % e)
+                print()
+                # @TODO: Temporary measure for the Beta program, until we settle
+                #        on a proper way to deal with authentication.
+                #        (c.f. discussion W JF Boismenu regarding Tank and DCCs)
+                sys.exit(-1)
 
             try:
                 try:
                     # Try to generate a session token and return the user info.
                     return hostname, login, session_cache.generate_session_token(
-                        hostname, login, password, http_proxy
-                    )
+                        hostname, login, password, http_proxy, None
```

Comment:
if you explicitly use the form `param_name=None` here the code is easier to read. Or maybe these None values can be defaults for the `generate_session_token` method?

Reply:
Actually, this is a left over from the early part of my work. The additional parameter is not needed anymore as it is optional for `generate_session_token` .